### DISCUSSION TITLE
Live Preview in Chrome/Edge stopped working

### DISCUSSION THREAD

**Discussion by john-do3** [phcode-dev/phoenix#1902] [General]
See "This site can’t provide a secure connection", "localhost sent an invalid response." ERR_SSL_PROTOCOL_ERROR page instead of site preview. How to fix this issue?

**Comment by abose** [comment]
This should be the browser app i am assuming. It may get fixed on restarting the browser or restarting the system. If that did not fix the issue, please comment here.

The desktop apps will not have this issue BTW and is recommended for pro users.

**Reply by john-do3** [reply]
Rebooting the system doesn't help. 
UPD: preview working OK in Firefox, and doesn't work in Chrome/Edge.

**Reply by abose** [reply]
Does preview work in incognito mode in chrome?

**Reply by john-do3** [reply]
No, incognito mode doesn't help.

**Reply by abose** [reply]
That is quite strange indeed. Is it working in any other system in the network from chrome?
